ui.py

```diff
@@ -10,9 +10,9 @@ def display_menu_get_choice(menu):
         choice = input('Enter choice? ')
 
         if type(choice) == str:
```

Comment:
There's no way for choice to not be a string. input return strings.

Reply:
Yes, you are right. Totally miss that one out. Thanks for pointing it out!